### PR TITLE
fix: add correct url for accepting invitation and remove JWT from accept endpoint

### PR DESCRIPTION
**Pull request by dipak-pawar**
**Changes Proposed in this PR**
* Removes Security from `api/accept/:acceptCode`
* Corrected accept url endpoint

### CODE REVIEW

File: authorization/invitation/service/invitation_service.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ func (s *invitationServiceImpl) processTeamInviteNotifications(ctx context.Conte
 	var messages []notification.Message
 
 	for _, n := range notifications {
-		acceptURL := fmt.Sprintf("%s/api/invitations/accept?code=%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
+		acceptURL := fmt.Sprintf("%s/api/invitations/accept/%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
```
Comment (xcoulon) [inline authorization/invitation/service/invitation_service.go]: can we have a constant for `/api/invitations/accept/`, so it does not need to be duplicated twice here and once more in https://github.com/fabric8-services/fabric8-auth/pull/607/files#diff-bdfa221aeccbdd7f5ee03b7c859c6c2dR47 ?

Comment (xcoulon) [reply authorization/invitation/service/invitation_service.go]: well done with the URL fix!

**Comment by fabric8cd** [comment]
@dipak-pawar snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-607-2`

File: authorization/invitation/service/invitation_service.go [hunk 2]
```diff
@@ -253,7 +255,7 @@ func (s *invitationServiceImpl) processTeamInviteNotifications(ctx context.Conte
 	var messages []notification.Message
 
 	for _, n := range notifications {
-		acceptURL := fmt.Sprintf("%s/api/invitations/accept?code=%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
+		acceptURL := fmt.Sprintf("%s%s%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
```
Comment (golangci) [inline authorization/invitation/service/invitation_service.go]: Sprintf format %s reads arg #3, but call has 2 args

**Pull request merged**